src/server.h

```diff
@@ -468,6 +481,8 @@ typedef enum {
 #define SLAVE_STATE_ONLINE 9 /* RDB file transmitted, sending just updates. */
 #define SLAVE_STATE_RDB_TRANSMITTED 10 /* RDB file transmitted - This state is used only for
                                         * a replica that only wants RDB without replication buffer  */
+#define SLAVE_STATE_BACKGROUND_RDB_LOAD 11 /* Main connection of a replica which uses rdb-channel-sync. */
```

Comment:
maybe this is more consistent and also clear?
```suggestion
#define SLAVE_STATE_HAS_ACTIVE_2ND_CHAN 11 /* Main connection of a replica which uses rdb-channel-sync. */
```